utility/src/main/java/org/oppia/android/util/logging/firebase/BUILD.bazel

```diff
@@ -0,0 +1,79 @@
+"""
+General purpose utilities for firebase logging.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "event_logger",
+    srcs = [
+        "FirebaseEventLogger.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
```

Comment:
This should be private--it's only needed for the module implementation. Ditto for all other non-modules packages here.

Note that in general for implementation packages like this we want to minimize visibility (generally only a module will be exposed for public access).

Also, I think you can just combine all of these into an "impl" since they aren't being exposed like other public utilities.

Reply:
@BenHenning thanks for explaining.

Reply:
Done